manifests/devopsconsole/0.1.0/devopsconsole-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,124 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    capabilities: Full Lifecycle
+    description: Enables the DevOps perspective in OpenShift 4.x
+    categories: "Developer Tools"
+  name: devopsconsole-operator.v0.1.0
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - kind: Component
+      name: components.devopsconsole.openshift.io
+      version: v1alpha1
+    - kind: GitSource
+      name: gitsources.devopsconsole.openshift.io
+      version: v1alpha1
+  description: The operator that enables the DevOps perspective in OpenShift 4.x
+  displayName: OpenShift DevOps Console
+  install:
+    spec:
+      deployments:
+      - name: devopsconsole-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: devopsconsole-operator
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                name: devopsconsole-operator
+            spec:
+              containers:
+              - command:
+                - devopsconsole-operator
+                env:
+                - name: WATCH_NAMESPACE
+                  valueFrom:
```

Comment:
We would like to use the `value` as "" for now.

Reply:
I am not getting this suggestion. Please clarify.

Reply:
@baijum I think what @sbose78 suggesting is no need to get namespace value using `valueFrom` just set it as `""`(empty string).

Reply:
Done. 5224d1a1be728f491091067f7b0e2eeb0d9a48c3